packages/patrol_cli/lib/src/features/test/ios_deploy.dart

```diff
@@ -0,0 +1,77 @@
+import 'dart:io';
+
+import 'package:dispose_scope/dispose_scope.dart';
+import 'package:file/file.dart';
+import 'package:patrol_cli/src/common/extensions/process.dart';
+import 'package:patrol_cli/src/common/logger.dart';
+import 'package:process/process.dart';
+
+class IOSDeploy {
+  IOSDeploy({
+    required ProcessManager processManager,
+    required FileSystem fs,
+    required DisposeScope parentDisposeScope,
+    required Logger logger,
+  })  : _processManager = processManager,
+        _fs = fs,
+        _disposeScope = DisposeScope(),
+        _logger = logger {
+    _disposeScope.disposedBy(parentDisposeScope);
+  }
+
+  final ProcessManager _processManager;
+  final FileSystem _fs;
+  final DisposeScope _disposeScope;
+  final Logger _logger;
+
+  /// Installs and launches the app on physical iOS device with [deviceId].
+  ///
+  /// To kill the app, send `kill` to the returned [Process]' stdin.
+  Future<Process> installAndLaunch(String deviceId) async {
+    final process = await _processManager.start(
+      [
+        'ios-deploy',
+        ...['--id', deviceId],
+        ...[
+          '--bundle',
+          '../build/ios_integ/Build/Products/Debug-iphoneos/Runner.app'
+        ],
+        // TODO: Enable deltas
+        // ...['--app_deltas', '../build/ios_integ/Runner-delta'],
+        '--debug',
+        '--no-wifi',
+        ...[
```

Comment:
Is that spread for clarity so that `--args` and the latter part are bound together?

Reply:
yes

makes the code easier to read